community/pmc-tool/main/views.py

```diff
@@ -1,8 +1,10 @@
 from django.shortcuts import render
 from main.collector.db import create_db_connection, create_pull_requests_table, insert_pull_requests_to_db, \
-    fetch_pull_requests, create_issues_table, fetch_issues, insert_issues_to_db
+    fetch_pull_requests, create_issues_table, fetch_issues, insert_issues_to_db, \
+    fetch_first_pr_authors_last_week, fetch_first_pr_date
 from main.collector.gh import query_prs, query_issues
 import logging
+import datetime
 
 
 def assert_equal(db_data, api_data):
```

Comment:
this function `assert_equal` does not make sense in this file. Please create a new file called `validations.py` and add that function there.